tezos/conv/src/to_ocaml.rs

```diff
@@ -108,6 +108,7 @@ impl_to_ocaml_variant! {
     ContextKvStoreConfiguration => OCamlContextKvStoreConfiguration {
         ContextKvStoreConfiguration::ReadOnlyIpc,
         ContextKvStoreConfiguration::InMem,
+        ContextKvStoreConfiguration::OnDisk,
```

Comment:
Oh, I need to add this to the OCaml side right? I guess that is why `InMem` points to the persistent implementation right now. I will add it tomorrow.